### PR TITLE
Adding compat-openssl11 to the builder

### PR DESCRIPTION
**Pull request by ParsaHarooni**
Fixes issue #12 and adds compat-openssl11 to the builder image.

### CODE REVIEW

File: builder/Dockerfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ RUN set -x; mkdir -p /go/src/ \
     && yum upgrade --refresh -y \
     && yum install -y \
         bc file findutils gpgme git hostname lsof make socat tar tree util-linux wget which zip \
-        gcc-toolset-12 go-toolset openssl openssl-devel \
+        gcc-toolset-12 go-toolset openssl openssl-devel compat-openssl11\
```
Comment (aleskandro) [inline builder/Dockerfile]:
```suggestion
        gcc-toolset-12 go-toolset openssl openssl-devel compat-openssl11 \
```

**Comment by aleskandro** [comment]
Hi @ParsaHarooni, thanks for your contribution!

Just a suggestion to fix a missing space...

Could you also update the title and the description of this PR (and its commit)?

**Comment by ParsaHarooni** [comment]
@aleskandro Sure, Fixed and updated the PR.

**Review by aleskandro — APPROVED** [review]
Thanks

**Pull request merged**